src/OSPSuite.Infrastructure.Import/Services/ICsvSeparatorSelector.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace OSPSuite.Infrastructure.Import.Services
+{
+   public interface ICsvSeparatorSelector
+   {
+      /// <summary>
+      /// Returns the separator to be used for the reading of the .csv file
+      /// </summary>
+      char GetCsvSeparator(string fileName);
```

Comment:
why not char?

Reply:
I like the suggestion for nullable types. I have changed it. Thanks Michael :)